specials/ladybug/build.xml

```diff
@@ -41,14 +41,14 @@
 		<property name="pom.xml" location="${basedir}/../../../${ff.dir}/ladybug/pom.xml" if:true="${test.with.iaf}"/>
 		<condition property="ladybug.version.ok">
 			<or>
-				<resourcecontains resource="${pom.xml}" substring="&lt;ladybug.version&gt;${ladybug.pom.xml.project.version}&lt;/ladybug.version&gt;"/>
+				<resourcecontains resource="${pom.xml}" substring="&lt;ladybug.version&gt;${ladybug.pom.xml.project.properties.revision}&lt;/ladybug.version&gt;"/>
```

Comment:
Misschien is er een risico met backwards compatibility. We starten met de Frank!Runner oude versies van het FF! op. Starten we er ook oude versies van Ladybug mee op? In dat geval moeten we de oude check met `ladybug.pom.xml.project.version` niet wegdoen maar ze allebei doen.

Is resource `pom.xml` hier de pom van ladybug-test-webapp?

Reply:
Het is inderdaad die mogelijk om een extra <or> toe te voegen, maar ik weet niet of we hier al te veel naar backwards compatibility moeten kijken. Wat denk jij @jacodg?

Reply:
Ik heb het er voor nu bij gedaan. Meld maar als je het anders wilt.